### DISCUSSION TITLE
Please fix problem with Linux Mint Mate sound preferences setting

### DISCUSSION THREAD

**Discussion by wpshooter** [community/community#133099] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Please fix the long standing problem in Linux Mint mate that when you attempt to set the sound preference/sound theme to the LINUX MINT choice, you have to do it twice (2 times) in order to get it to take/save that setting.  When you change the sound theme setting to Linux Mint the first time it appears to not save it.  Then you have to go back and choose the Linux Mint from the drop down the SECOND TIME before it will actually change the theme setting to Linux Mint.  Thanks.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @wpshooter ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ebndev** [comment]
Hi @wpshooter, thanks for being a part of the GitHub Community!

It looks like you're looking for support from [Linux Mint](https://github.com/orgs/linuxmint/discussions). You might want to try asking for posting this in their repo's Discussions, as this is a public forum where the creators are unlikely to see your comment.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
